diff --git a/dist/pipelines-base-url.js b/dist/pipelines-base-url.js
index 383e480c1938d0169343aa7b3c39d7edc9357eb0..c4ca4b1d956d90d5564ce5a946f808cb05c61a35 100644
--- a/dist/pipelines-base-url.js
+++ b/dist/pipelines-base-url.js
@@ -6,13 +6,14 @@ export function setPipelinesBaseUrl(baseUrl) {
     pipelinesBaseUrl = baseUrl;
 }
 export function getPipelinesBaseUrl() {
-    if (typeof pipelinesBaseUrl !== "undefined") {
-        return pipelinesBaseUrl;
-    }
-    const itkWasmPipelinesBaseUrl = itkWasmGetPipelinesBaseUrl();
-    if (typeof itkWasmPipelinesBaseUrl !== "undefined") {
-        return itkWasmPipelinesBaseUrl;
-    }
-    return defaultPipelinesBaseUrl;
+    return new URL(`./pipelines`,import.meta.url)
+    // if (typeof pipelinesBaseUrl !== "undefined") {
+    //     return pipelinesBaseUrl;
+    // }
+    // const itkWasmPipelinesBaseUrl = itkWasmGetPipelinesBaseUrl();
+    // if (typeof itkWasmPipelinesBaseUrl !== "undefined") {
+    //     return itkWasmPipelinesBaseUrl;
+    // }
+    // return defaultPipelinesBaseUrl;
 }
 //# sourceMappingURL=pipelines-base-url.js.map
